README.md

```diff
@@ -184,6 +184,28 @@ go run widgets/segmentdisplay/segmentdisplaydemo/segmentdisplaydemo.go
 
 [<img src="./doc/images/segmentdisplaydemo.gif" alt="segmentdisplaydemo" type="image/gif">](widgets/segmentdisplay/segmentdisplaydemo/segmentdisplaydemo.go)
 
+## The TreeView
```

Comment:
Please merge from the `devel` branch and resolve the conflict.

Reply:
Will this require a new PR? [keithknott26](https://github.com/keithknott26) wants to merge 3 commits into [mum4k:devel](https://github.com/mum4k/termdash/tree/devel) I thought i was merging to devel